### PR TITLE
OLS-777: Document how to install the Lightspeed Operator

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): Tech Preview

Issue: https://issues.redhat.com/browse/OLS-777

Link to docs preview: https://78216--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/install/ols-installing-openshift-lightspeed.html#ols-installing-operator

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by thoraxe** [comment]
LGTM for now

**Comment by rh-tokeefe** [comment]
@JoaoFula when you have a moment, please provide review. Thanks!

File: modules/ols-installing-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later. The cluster must be connected to the Internet and have telemetry enabled.
+
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+
+* You have access to the {ocp-short-name} CLI (oc).
+
+* You have successfully configured your Large Language Model (LLM) provider so that OpenShift Lightspeed can commnuicate with it.
```
Comment (JoaoFula) [inline modules/ols-installing-operator.adoc:14]: typo on communicate

Comment (rh-tokeefe) [reply modules/ols-installing-operator.adoc]: Done. Updated PR. Thank you for catching this!

**Comment by JoaoFula** [comment]
LGTM

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**